pybamm/models/submodels/thermal/base_thermal.py

```diff
@@ -216,6 +229,7 @@ def _get_standard_coupled_variables(self, variables):
         Q_ohm_vol_av = Q_ohm_W / V
         Q_rxn_vol_av = Q_rxn_W / V
         Q_rev_vol_av = Q_rev_W / V
+        Q_cr_vol_av = Q_cr_W / V
```

Comment:
I think this term should be `I^2R/V` and we need to reverse engineer the previous terms to get to that. i.e. `Q_cr = I**2 / (n_elec * A) * R_contact`. Need input from @rtimms @brosaplanella @TomTranter about how to think about contact resistance heating in "volume-averaged" sense

Reply:
I'm not sure the contact resistance heating makes sense outside of the lumped model - it's not obvious "where" the heating is taking place. For higher-dimensional models, it's certainly not the case that the contact resistance heating should appear as a uniform source everywhere in the cell.